src/components/inputs/z-select/styles.css

```diff
@@ -82,3 +82,8 @@
 .select-wrapper .ul-scroll-wrapper .no-results z-icon {
   margin-right: var(--space-unit);
 }
+
+.z-list-group-title {
```

Comment:
Questa cosa avevi poi chiesto a Pier? In alternativa si potrebbero fare due variabili css che gestiscono questi stili se devono essere parametrizzabili

Reply:
La grafica allegata alla storia va bene, modifico lo stile con quello nuovo che mi ha passato. Va bene renderlo unico e cablarlo nel componente